config_templates/gretel/synthetics/tabular-differential-privacy.yml

```diff
@@ -0,0 +1,12 @@
+# Default configuration for Gretel Tabular DP to generate synthetic data with
+# differential privacy guarantees
+
+schema_version: "1.0"
+name: "tabular-differential-privacy"
+models:
+  - tabular_dp:
+      data_source: __tmp__
+      params:
+        epsilon: 1
+        delta: 0.0001
+        infer_domain: True
```

Comment:
I'd expect to see a newline at the end of the file (I think other yml files here do that, though a little hard to tell). Unless a VSCode or other autoformatter is removing a trailing newline automatically.